lib/python/eupath/GeneListHandler.py

```diff
@@ -53,13 +54,21 @@ def create_formatted_genelist_file(self):
         valid_sep_exp = r"[\s,;]"
         formatted_file = open('formatted_genelist.txt', 'w')
         first = True
+        genes_set = {"initialize_this_set_with_something"}
```

Comment:
Maybe `genes_set = set()` here? I'm guessing the init value is just to ensure it's a set and not a dict.